docker/artemis.yml

```diff
@@ -0,0 +1,45 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Artemis base service
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    artemis-app:
+        container_name: artemis-app
+        platform: linux/amd64
+        # look in the docs for more information about how to build your Artemis code with Docker
+        # https://docs.artemis.cit.tum.de/dev/setup/#
+        image: ghcr.io/ls1intum/artemis
+        build:
+            context: ..
+            dockerfile: docker/artemis/Dockerfile
+            pull: true
+            target: runtime
+        volumes:
+            - artemis-data:/opt/artemis/data
+        env_file:
+            - ./artemis/config/prod-application-local.env
+        # environments can also be used for custom overrides (alternative to application-local.yml)
+        # the following environments are necessary for docker images orchestrated by docker compose
+        environment:
+            SPRING_PROFILES_ACTIVE: artemis,scheduling,athene,prod,docker
```

Comment:
Is the docker profile new?

Reply:
Yes, it contains default configs for all docker setups so that you don't have to define them over and over again, like the DB connection setting it's default to ``artemis-mysql`` instead of ``localhost``. So all ``.yml`` files in the config folder except ``application-local.*.yml`` define default configs values which can later be overwritten with more specific config values. For these override values we use ENV files instead of an ``application-local.yml`` file. One example where this is done is the [cypress setup](https://github.com/ls1intum/Artemis/blob/develop-deployment-wg/docker/cypress-E2E-tests.yml).